src/commands/db/backup/download.ts

```diff
@@ -0,0 +1,115 @@
+import ora from 'ora';
+import inquirer from 'inquirer';
+import Command from '../../../base.js';
+import { Args, Flags, ux } from '@oclif/core';
+import { createDebugLogger } from '../../../utils/output.js';
+import IGetDatabasesResponse from '../../../types/get-dbs-response.js';
+import * as shamsi from 'shamsi-date-converter';
+
+export interface IBackUp {
+  name: string;
+  lastModified: string;
+  etag: string;
+  size: number;
+}
+
+export interface IBackups {
+  backups: IBackUp[];
+}
+
+export default class BackUp extends Command {
+  static description = 'download a database backup';
+
+  static PATH = 'v1/databases/{database-id}/backups';
+
+  static aliases = ['db:backup:dl'];
+
+  static flags = {
+    ...Command.flags,
+    name: Flags.string({
+      char: 'n',
+      description: 'name of your database',
+    }),
+    backup: Flags.string({
+      char: 'b',
+      description: 'select which backup to download',
+    }),
+  };
+
+  async run(): Promise<void> {
+    this.spinner = ora();
+
+    const { flags, args } = await this.parse(BackUp);
+    const debug = createDebugLogger(flags.debug);
+
+    await this.setGotConfig(flags);
+    const account = await this.getCurrentAccount();
+
+    ((account && account.region === 'germany') || flags.region === 'germany') &&
+      this.error('We do not support germany any more.');
+
+    const hostname = flags.name || (await this.promptHostname());
+
+    try {
+      const database = await this.getDatabaseByHostname(hostname);
+      if (database === undefined) {
+        this.log(`Database ${hostname} not found`);
+        return;
+      }
+      const databaseID = database._id;
+      const backupName = flags.backup || (await this.promptBackupName());
+      const downloadLink = await this.got
+        .post(
+          BackUp.PATH.replace('{database-id}', databaseID) +
+            `/${backupName}/download`
+        )
+        .json<any>();
+      this.log(`download link: ${downloadLink.link}`);
```

Comment:
I think this command should download the backup to the current working directory.